style.css

```diff
@@ -589,6 +613,162 @@ body {
   margin-top: 114px;
 }
 
+/* popup window */
+.popup .popup-window {
+  width: 90%;
+  background-color: #fff;
+  margin: 0 auto;
+  border: 1px solid #ebecf0;
+  border-radius: 5px;
+  padding: 0 20px;
+}
+
+.all-details {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+}
+
+.tonic-popup {
+  color: #091e42;
+  font-size: 35px;
+  font-weight: 700;
+  line-height: 30px;
+  letter-spacing: 0;
+  margin-top: 30px;
+}
+
+.canop-popup {
+  display: flex;
+  flex-flow: row wrap;
+  margin-top: 15px;
+}
+
+.can-info-popup {
+  color: #344563;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.point-popup {
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  border-radius: 5px;
+  margin: 3px 0 0 6px;
```

Comment:
You did well with the modal popup, kindly note that you are expected to follow the design template for both mobile and desktop version of your display 👍 , so please work on the mobile version to be similar to the figma design

your design :


https://github.com/mar4444/re-myportfolio1/assets/20537712/0e126e81-428d-404b-a39c-df4137f27212